docs/content/en/docs/implementing/tasks-non-k8s-apps.md

```diff
@@ -51,10 +51,6 @@ kind: KeptnTask
 metadata:
   name: runhelloworld1
 spec:
-  workload: "my-workload"
```

Comment:
I know this is very rude of me to ask, but since we have to change this, could we update the samples instead and embed the yamls ref here? 😇

Reply:
sure, will update that today